cosmos/azure-spring-data-cosmos/spring3-samples/cosmos-aad-sample/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../../pom.xml</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>cosmos-spring-aad-sample</artifactId>
+  <version>1.0.0</version>
+
+  <properties>
+    <spring.boot.version>2.7.1</spring.boot.version>
+    <azure.spring.data.cosmos.version>3.29.1</azure.spring.data.cosmos.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-spring-data-cosmos</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+    </dependency>        
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+    </dependency>
+  </dependencies>
+</project>
```

Comment:
need a new line

Reply:
Do you mean at the end of the file? Why is a new line needed there?

Reply:
yes, I think it is better for the format to align with others.

Reply:
or it will show this symbol 
![image](https://github.com/Azure-Samples/azure-spring-boot-samples/assets/92105726/dc1bf5a3-c9e5-4dae-bfb8-65b51a622f18)